#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_no_examples.dpatch by  <Mike Hommey <glandium@debian.org>>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Install the files for about:plugins. bz#328678

@DPATCH@

=== toolkit/content/jar.mn
==================================================================
--- xulrunner/toolkit/content/jar.mn	(revision 63)
+++ xulrunner/toolkit/content/jar.mn	(local)
@@ -74,3 +74,4 @@
 #ifdef MOZ_SVG
    content/global/svg/svgBindings.xml          (/layout/svg/base/src/resources/content/svgBindings.xml)
 #endif
+   content/global/plugins.html                 (../../xpfe/global/resources/content/plugins.html)
=== toolkit/themes/winstripe/global/jar.mn
==================================================================
--- xulrunner/toolkit/themes/winstripe/global/jar.mn	(revision 65)
+++ xulrunner/toolkit/themes/winstripe/global/jar.mn	(local)
@@ -114,3 +114,4 @@
         skin/classic/global/tree/twisty-open.png                    (tree/twisty-open.png)
         skin/classic/global/throbber/Throbber-small.gif             (throbber/Throbber-small.gif)
 	skin/classic/global/throbber/Throbber-small.png             (throbber/Throbber-small.png)
+        skin/classic/global/plugins.css                             (../../../../themes/classic/global/plugins.css)
